File: Tareas/Unidad2/a2253332351_practica04/35_LitrosAGalones.java
import java.util.Scanner;

public class LitrosAGalones {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double litros;
        double galones;

        System.out.print("Ingrese la cantidad de litros: ");
        litros = entrada.nextDouble();

        galones = litros * 0.264172;

        System.out.println("La cantidad equivalente en galones es: " + galones);
    }
}

File: Tareas/Unidad2/a2253332351_practica04/35_LitrosAGalones.cpp
#include <iostream>
using namespace std;

int main() {
    double litros;
    double galones;

    cout << "Ingrese la cantidad de litros: ";
    cin >> litros;

    galones = litros * 0.264172;

    cout << "La cantidad equivalente en galones es: " << galones << endl;

    return 0;
}
